src/main/java/crazypants/enderio/machine/farm/FakeFarmPlayer.java

```diff
@@ -88,4 +89,12 @@ public void func_147100_a(C15PacketClientSettings pkt) {
   public boolean canPlayerEdit(int par1, int par2, int par3, int par4, ItemStack par5ItemStack) {
     return true;
   }
+
+  @Override
+  public void setWorld(World p_70029_1_) {
+    Log.warn("Ender IO Farming station fake player is being transfered to world '" + p_70029_1_
+        + "'. Trying to reject transfer. Call stack follows:");
+    (new Exception()).printStackTrace();
```

Comment:
Heh, I didn't actually read the source, but the it's more about the principle, no? :P